### PR TITLE
Updated the picture for Ikea E1766 with a higher quality one

### PR DESCRIPTION
**Pull request by careyer**
old pictures had a black background and fingers in it..
This one has a clean white background

### CODE REVIEW

File: docs/devices/E1766.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ description: "Integrate your IKEA E1766 via Zigbee2MQTT with whatever smart home
 | Vendor  | IKEA  |
 | Description | TRADFRI open/close remote |
 | Exposes | battery, action, linkquality |
-| Picture | ![IKEA E1766](../images/devices/E1766.jpg) |
+| Picture | ![IKEA E1766](../images/devices/E1766_2.jpg) |
```
Comment (Koenkk) [inline docs/devices/E1766.md:16]: Please replace the old E1766.jpg instead of adding a new image

Comment (careyer) [reply docs/devices/E1766.md]: Alright! I have replaced the old E1766,jpg with the new one! 
Just did not want to destroy somebody elses work ;-)

**Comment by careyer** [comment]
Done! Replaced old file with new. Cheers!

**Comment by Koenkk** [comment]
Thanks!

**Comment by careyer** [comment]
You are welcome! Keep up the terrific work! Zigbee2mqtt rulez! 🤘🏻

Von meinem iPhone gesendet

**Pull request merged**